### ISSUE TITLE
xdebug not working for generic-php environment

### ISSUE THREAD

**Issue by OliverBeckwith** [rewardenv/reward#40]
I have a `generic-php` environment that I want to debug with xdebug. I have followed the steps found in https://rewardenv.readthedocs.io/en/latest/configuration/xdebug.html#debugging-xdebug-instance, however when making API calls with the `XDEBUG_SESSION=PHPSTORM` parameter, no activity is logged from the debug container.

This is my `.env` file if it helps:
```
REWARD_ENV_NAME=api-php
REWARD_ENV_TYPE=generic-php
REWARD_WEB_ROOT=/

TRAEFIK_DOMAIN=api-php.test
TRAEFIK_SUBDOMAIN=
TRAEFIK_EXTRA_HOSTS=

REWARD_DB=1
REWARD_REDIS=1
REWARD_VARNISH=1

MARIADB_VERSION=10.4
NODE_VERSION=10
PHP_VERSION=8.0
REDIS_VERSION=6.0
COMPOSER_VERSION=2
VARNISH_VERSION=7.1

MYSQL_EXPOSE=true
MYSQL_ROOT_PASSWORD=root
MYSQL_DATABASE=database
MYSQL_USER=root
MYSQL_PASSWORD=root

NGINX_ROOT=/var/www/html
NGINX_PUBLIC=

XDEBUG_VERSION=3
```

**Comment by janosmiko** [comment]
Hi @OliverBeckwith ,

Could you provide more details about the issue?

I tried to reproduce it, but to me, it works out of the box.

I did the following:
```
mkdir ~/Workspace/tmp/generic
cd ~/Workspace/tmp/generic

reward env-init --environment-type generic-php generic

reward env up
```

I created an `index.php` file with the following content and I added a breakpoint to the var_dump() function in VSCode:
```php
<?php
var_dump($_GET);
phpinfo();
?>
```

I modified the default debug configuration. (The only change I made was adding the pathMappings.)
`launch.json`
```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Listen for Xdebug",
            "type": "php",
            "request": "launch",
            "port": 9003,
            "pathMappings": {
                "/var/www/html": "${workspaceFolder}"
            }
        }
    ]
}
```

And when I enabled the Xdebug cookie in Chrome Xdebug Helper plugin it instantly worked:
<img width="502" alt="image" src="https://user-images.githubusercontent.com/693558/233928794-5e55345d-ae07-4e0a-803a-022312c85aab.png">

Nginx uses the following configuration to route the traffic to the Xdebug container:
```
# Select upstream backend to use based on presence of Xdebug cookies and Blackfire headers
map "$http_X_BLACKFIRE_QUERY:$cookie_XDEBUG_SESSION$cookie_XDEBUG_PROFILE$cookie_XDEBUG_TRACE" $fastcgi_backend {
    # Nothing for debug and nothing for blackfire means its a pure request
    ":" {{ getenv "NGINX_UPSTREAM_HOST" "php-fpm" }}:{{ getenv "NGINX_UPSTREAM_PORT" "9000" }};

    # Use blackfire if the blackfire query is specified AND no debug cookie is set
    "~:$" {{ getenv "NGINX_UPSTREAM_BLACKFIRE_HOST" "php-blackfire" }}:{{ getenv "NGINX_UPSTREAM_BLACKFIRE_PORT" "9000" }};

    # In all other cases, a debug cookie will be present; use debug container
    default {{ getenv "NGINX_UPSTREAM_DEBUG_HOST" "php-debug" }}:{{ getenv "NGINX_UPSTREAM_DEBUG_PORT" "9000" }};
}
```
Basically, if any of the `XDEBUG_SESSION`, `XDEBUG_PROFILE` or `XDEBUG_TRACE` cookies are set the request will be sent to the Xdebug container.

I was also able to trigger debug session using this curl:
```
curl "http://generic.test/" \
     -H 'Cookie: XDEBUG_SESSION=PHPSTORM'
```
